src/test/java/uk/gov/hmcts/reform/et/syaapi/service/NotificationServiceTest.java

```diff
@@ -540,5 +543,97 @@ void shouldNotSendResponseEmailToClaimantForTypeCApplication() throws Notificati
             eq(testData.getExpectedDetails().getId().toString())
         );
     }
+
+
+    @Test
+    void shouldSendResponseEmailToRespondent() throws NotificationClientException {
+        notificationService.sendResponseEmailToRespondent(
+            testData.getCaseData(),
+            CLAIMANT,
+            "1",
+            TEST_RESPONDENT,
+            NOT_SET,
+            testData.getExpectedDetails().getId().toString(),
+            CHANGE_DETAILS_APPLICATION_TYPE
+        );
+
+        verify(notificationClient, times(1)).sendEmail(
+            any(),
+            eq(testData.getCaseData().getRespondentCollection().get(0).getValue().getRespondentEmail()),
+            any(),
+            eq(testData.getExpectedDetails().getId().toString())
+        );
+    }
+
+    @Test
+    void shouldSendResponseEmailToRespondentResp() throws NotificationClientException {
+        RespondentSumType respondentSumType = new RespondentSumType();
+        respondentSumType.setRespondentEmail("test@resRep.com");
+
+        RespondentSumTypeItem respondentSumTypeItem = new RespondentSumTypeItem();
+        respondentSumTypeItem.setValue(respondentSumType);
+        respondentSumTypeItem.setId(String.valueOf(UUID.randomUUID()));
+
+        CaseData caseData = testData.getCaseData();
+        caseData.getRespondentCollection().add(respondentSumTypeItem);
+
+        notificationService.sendResponseEmailToRespondent(
+            testData.getCaseData(),
+            CLAIMANT,
+            "1",
+            TEST_RESPONDENT,
+            NOT_SET,
+            testData.getExpectedDetails().getId().toString(),
+            CHANGE_DETAILS_APPLICATION_TYPE
+        );
+
+        verify(notificationClient, times(1)).sendEmail(
+            any(),
+            eq("test@resRep.com"),
+            any(),
+            eq(testData.getExpectedDetails().getId().toString())
+        );
+    }
+
+    @Test
+    void shouldNotSendResponseWhenRespondentEmailDoesNotExist() throws NotificationClientException {
```

Comment:
Why do we have a `throws` on these tests?

Reply:
The send email function throws and NotificationClientException(just like other tests)